woocommerce/woocommerce-admin
# Add initial documentation for the historical data import.

Fixes #3370.

Adds documentation about the historical data import.

### Detailed test instructions:

- Check the formatting and read the doc
- Check for accuracy
